### PR TITLE
chore: remove SVGs from a11y tree

### PR DESCRIPTION
**Pull request by ogonkov**

### CODE REVIEW

File: src/components/utils/svgA11yHidden.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const svgA11yHidden = {
+    // Hides element from a11y tree
+    'aria-hidden': true,
+    // Prevents focus in IE
+    focusable: false,
```
Comment (amje) [inline src/components/utils/svgA11yHidden.ts:5]: We don't support IE, so there is no need to add this.

File: src/components/utils/svgA11yHidden.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const svgA11yHidden = {
```
Comment (amje) [inline src/components/utils/svgA11yHidden.ts:1]: Not sure about standalone file and object itself cause we have only one prop needed (`aria-hidden`)

File: src/components/utils/svgA11yHidden.ts (hunk 1 above)
Comment (ogonkov) [reply src/components/utils/svgA11yHidden.ts:5]: I guess it is not a big deal, we improve UX for IE users, they will receive redundant focus instead

Comment (amje) [reply src/components/utils/svgA11yHidden.ts:5]: Redundant focus will be the least problem. JS simply won't work in IE due to modern features we are using in the code. No support for IE - no legacy code

File: src/components/utils/svgA11yHidden.ts (hunk 2 above)
Comment (ogonkov) [reply src/components/utils/svgA11yHidden.ts:1]: Code itself is clear about it purpose, that could be not clear why svgs receive `aria-hidden`

Comment (amje) [reply src/components/utils/svgA11yHidden.ts:1]: Ok, but let's name it accordingly, `a11yHiddenSvgProps`, more accent on props. And also let's rename file to `utils/svg.ts` for future svg code

Comment (ogonkov) [reply src/components/utils/svgA11yHidden.ts]: Done

**Review by amje — APPROVED** [review]

**Pull request merged**